File: shared-library/src/types/voice.ts
import { AvatarGender } from './avatar';

export type Voice = {
    id: string;
    name: string;
    gender: AvatarGender;
    age: string;
    category: string;
    description: string;
    language: string;
    previewUrl: string;
    useCase: string;
}

File: shared-package-py/loom24_shared/types/voice.py
from __future__ import annotations

from pydantic import BaseModel

from .avatar import AvatarGender


class Voice(BaseModel):
    id:         str
    name:       str
    gender:     AvatarGender
    age:        str
    category:   str
    description: str
    language:   str
    previewUrl: str
    useCase:    str
